src/screens/settings/settings.tsx

```diff
@@ -119,6 +123,20 @@ export const Settings = () => {
               <WhiteContainerText text="About" />
               <Icon name={IconNameEnum.ChevronRight} size={formatSize(24)} />
             </WhiteContainerAction>
+            <WhiteContainerDivider />
+            <WhiteContainerAction
+              onPress={() => {
+                trackEvent(SettingsSelectors.Share, AnalyticsEventCategory.ButtonPress);
+                Share.share({
```

Comment:
if `Share` returns `Promise` try to send into analytics successful share & share rejection

Reply:
ok, good point